src/js/components/Select/Select.js

```diff
@@ -169,6 +173,10 @@ const Select = props => {
         margin={margin}
         onOpen={onRequestOpen}
         onClose={onRequestClose}
+        onMouseOver={onMouseOver}
+        onMouseOut={onMouseOut}
+        onBlur={onBlur}
+        onFocus={onFocus}
```

Comment:
Is there a reason that we don't apply `...rest` to this element, instead of the SelectTextInput? That would have covered the need.

Reply:
hm, i am assuming for styling purposes of the text input although i haven't checked this a lot.